purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterSurverOptionChosenEvent.kt

```diff
@@ -0,0 +1,54 @@
+package com.revenuecat.purchases.customercenter.events
+
+import androidx.annotation.VisibleForTesting
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.common.events.FeatureEvent
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.json.Json
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Type representing a customer center event and associated data. Meant for RevenueCatUI use.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+@Poko
+class CustomerCenterSurverOptionChosenEvent(
+    val creationData: CreationData = CreationData(),
+    val data: Data,
+) : FeatureEvent {
+
+    companion object {
+        @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+        internal val json = Json.Default
```

Comment:
I believe this was mentioned in another PR, but ideally we reuse the same json serializer/deserializer accross the app. Otherwise we might forget some setting. For example, we might want to set `ignoreUnknownKeys = true` so we don't have any issues with unknown keys...